docker-compose.yml

```diff
@@ -1,8 +1,34 @@
 version: '3.9'
 
+volumes:
+  postgres:
+
 services:
-    entropay-employees:
-      build: .
-      ports:
-          - "8100:8100"
+  app:
+    build: .
+    env_file: local.env
+    ports:
+        - "8100:8100"
+
+  postgres:
+    image: 'postgres:10.2-alpine'
+    environment:
+      - 'POSTGRES_DB=employees'
```

Comment:
do we have an story to encrypt credentials?

Reply:
These are the credentials for a local db, i think we don't need encrypt those